typhos/__init__.py

```diff
@@ -16,12 +7,26 @@
 from .suite import TyphosSuite
 from .utils import load_suite, patch_connect_slots, use_stylesheet
 
+__all__ = [
+    'use_stylesheet',
+    'register_signal',
+    'load_suite',
+    'TyphosCompositeSignalPanel',
+    'TyphosDeviceDisplay',
+    'TyphosSuite',
+    'TyphosSignalPanel',
+    'TyphosPositionerWidget',
+    'TyphosMethodButton',
+    '__version__',
```

Comment:
```suggestion
```

`from typhos import *` probably shouldn't include `__version__`

Reply:
Looks like this has been sneakily hiding here for about 2 years
Funny how unrelated diffs can bring stuff like this to our attention

Reply:
This also underscores some of the advantages of these style changes. This list is must easier to read when pulled out vertically like this rather than partially wrapped into multiple entries on some lines.

Reply:
Totally agreed!

I sometimes complain about how `black` takes up too much vertical space. This sort of thing shows me that I should probably get over it and start adopting its style more often.